2012-10-19  Florian Pritz  <bluewind@xinu.at>

	[BZ #13758]
	* locales/de_AT (LC_TIME): Change month name from "FebruAr" to
	"Februar".

--- a/localedata/locales/de_AT
+++ b/localedata/locales/de_AT
@@ -102,7 +102,7 @@
         "<U0053><U0065><U0070>";"<U004F><U006B><U0074>";/
         "<U004E><U006F><U0076>";"<U0044><U0065><U007A>"
 mon     "<U004A><U00E4><U006E><U006E><U0065><U0072>";/
-        "<U0046><U0065><U0062><U0072><U0075><U0041><U0072>";/
+        "<U0046><U0065><U0062><U0072><U0075><U0061><U0072>";/
         "<U004D><U00E4><U0072><U007A>";/
         "<U0041><U0070><U0072><U0069><U006C>";/
         "<U004D><U0061><U0069>";/
